.github/workflows/.templates/build_packages.yaml

```diff
@@ -0,0 +1,502 @@
+{% if not slim %}
+name: Cross build packages
+{% else %}
+name: Build slim packages
+{% endif %}
+
+on:
+{% if not slim %}
+  schedule:
+    - cron:  '0 */6 * * *'
+  push:
+    tags:
```

Comment:
The slim build also needs to be triggered by push tag, because the complete build takes too long, and testers need to quickly obtain the installation package